app/src/main/res/layout/fragment_example.xml

```diff
@@ -1,95 +1,100 @@
 <?xml version="1.0" encoding="utf-8"?>
-<layout
+<layout xmlns:tools="http://schemas.android.com/tools"
 	xmlns:android="http://schemas.android.com/apk/res/android"
 	xmlns:app="http://schemas.android.com/apk/res-auto">
 
 	<androidx.constraintlayout.widget.ConstraintLayout
 		android:layout_width="match_parent"
 		android:layout_height="match_parent"
+		android:background="#FFFFFF"
```

Comment:
There's a file where you can save colors and make reference of them. Please, see the resources folder.